static/js/lib/util.js

```diff
@@ -2,6 +2,7 @@
 import type { Match } from "react-router"
 
 export const getChannelName = (props: { match: Match }): string => props.match.params.channelName || ""
+export const getPostID = (props: { match: Match }): string => props.match.params.postID || ""
```

Comment:
safer in what regard? I don't think we need to be quite that safe here, because if we don't have that data here it's a programmer error and we'd want it to fail hard. The `|| ""` is only here to quiet flow since it doesn't know better.